app/src/main/res/menu/activity_main_drawer.xml

```diff
@@ -9,5 +9,7 @@
             android:title="@string/tip_out"/>
         <item android:id="@+id/CountIn"
             android:title="@string/count_in"/>
+        <item android:id="@+id/settings"
```

Comment:
You id here needs to match you destination id in the main_navigation.xml file or this isn't going to work properly.